src/mesher/Mesher.cpp

```diff
@@ -0,0 +1,77 @@
+#include "gmsh.h"
+#include "Mesher.h"
+#include <string>
+
+
+Mesher::Mesher() { gmsh::initialize(); }
+Mesher::~Mesher(){ gmsh::finalize(); }
+
+//Options: dimensions: digit 1-3, myName: "filename.format"
+void Mesher::set_options(int dimensions, std::string myName)
```

Comment:
I think this could be confusing. Why do you just set two of the three values of the options? Why not pass an instance of `Mesher::Options` to this function?

Reply:
There were uncertainties regarding the size which are solved now. So the structure has changend.